suites/reef/cephadm/tier2-cluster-elasticity.yaml

```diff
@@ -603,12 +570,12 @@ tests:
         pool_type: 'normal'
       desc: run rados bench for 360 - normal profile
 
-  - test:
```

Comment:
why this test case is skipped ?

Reply:
rbd-io was throwing socket timeout at times. so commented it.